Gruntfile.js

```diff
@@ -0,0 +1,5 @@
+module.exports = function(grunt) {
+  'use strict';
+  grunt.loadNpmTasks('grunt-fh-build');
+  grunt.registerTask('default', ['eslint']);
```

Comment:
I'm not sure if we should use full `grunt-fh-build` if we're only going to use a couple of its tasks. I think it might be too focused on grouping tasks required for the modules for core and mbaas (on `grunt fh:default`, that's why a lot of core modules still depend on `grunt-fh-build < 1.0` because it introduced `eslint` on the default task), and raincatcher might diverge further and further from it.

For instance I noticed that there are some custom tools like [`raincatcher-template-build`](https://github.com/feedhenry-raincatcher/raincatcher-template-build) that are current used as standalone grunt tasks, would we move it inside `grunt-fh-build` or invoke it separately?